build/package/Microsoft.DotNet.Cli.Nupkg.targets

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <Target Name="SetupGenerateNugetPackagesInputsOutputs"
+          DependsOnTargets="Init">
+    <PropertyGroup>
+      <NupkgVersionSuffix>@(BuildVersion -> '%(CommitCount)')</NupkgVersionSuffix>
+      <ProjectsSrcDirectory>$(RepoRoot)/src</ProjectsSrcDirectory>
+      <PackagingBuildBasePath>$(Stage2CompilationDirectory)/forPackaging</PackagingBuildBasePath>
+      <NupkgOutputDirectory>$(BaseOutputDirectory)/packages</NupkgOutputDirectory>
```

Comment:
This can use the `<PackagesDirectory>$(BaseOutputDirectory)/packages</PackagesDirectory>` from the Init task.